### ISSUE TITLE
Modals on manage users page don't return focus to triggering element on close

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4103]
## Area or Component
Manage users modals (add user, edit email, edit name, etc)

## Quick Summary
Closing the modal returns keyboard focus back to the HTML`<body>`, instead of to the specific button that triggered the modal.

## Accessibility Violation
Modal is closed, focus is not returned to trigger: https://cdc-devtools.dequecloud.com/rule-help/en/focus-modal-not-returned

## Action Requested
Return the modal focus to the triggering button.

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1647999156&range=16:16

**Comment by georgehager** [comment]
I have been unable to reproduce this issue. When I tab-enter or click on each modal and close it, it returns focus to the triggering button. See the following:

https://user-images.githubusercontent.com/89797785/191591859-5c3f96ae-0c8f-4f67-97a1-58b592a39a04.mov

https://user-images.githubusercontent.com/89797785/191591884-c60e49a3-7f0f-4acd-8721-e637b5eca694.mov

**Comment by georgehager** [comment]
According to the [react-modal docs]():

>By default, when the modal is closed, focus will be restored to the element that was focused before the modal was opened. To disable this behavior, you can pass the shouldReturnFocusAfterClose={false} prop to your modal

We do not disable this behavior, so by default `react-modal` enables the behavior to focus on the element that triggered the modal when the modal is closed.

Confirmed that this is a false positive, most likely triggered by Deque's browser automation behavior since the same behavior is not observable when performing the same actions manually in the browser.

cc: @emmastephenson
